.travis.yml

```diff
@@ -11,7 +11,7 @@ notifications:
 sudo: false
 script:
     - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-    - julia -e 'Pkg.clone(pwd()); Pkg.build("JSON"); Pkg.test("JSON"; coverage=true)';
+    - julia -e 'Pkg.checkout("Compat"); Pkg.clone(pwd()); Pkg.build("JSON"); Pkg.test("JSON"; coverage=true)';
```

Comment:
it won't be installed yet, you'd have to do this after cloning JSON from pwd

not a bad idea re: submitting `tryparse` to Compat, not sure why no one has asked for it yet but if it's useful here then it's bound to be useful somewhere else